### PR TITLE
Foundations content update

### PR DESCRIPTION
**Pull request by jordanmccullough**
Front-to-back review of all slides, notes, titles, file names. Improves examples and clarity of content for both presentation purposes, as well as independent study/review.

cc: @github/training-team

### CODE REVIEW

**Comment by matthewmccullough** [comment]
Reviewing...

File: presentations/_posts/foundations/config/0002-01-01-Scopes.md [hunk 1]
```diff
@@ -4,14 +4,14 @@ layout: slide
...
+	git config --local <name> <value>
 
 __user__ configuration
 
-	git config --global --list
+	git config --global <name> <value>
 
 __system__ configuration
 
-	git config --system --list
```
Comment (matthewmccullough) [inline presentations/_posts/foundations/config/0002-01-01-Scopes.md:17]: What about reversing the order of these? I like system on top and local on bottom as it follows the imagery of inheritance a little better.

File: presentations/_posts/foundations/config/0002-02-01-Config-Scopes-Visualization.md [hunk 2]
```diff
@@ -1,8 +0,0 @@
----
-chapter: Config
-layout: slide
-tags: ['config']
----
-<div class="diagram-group">
-<img class="diagram" src="assets/diagrams/git-config-layers.png">
```
Comment (matthewmccullough) [inline presentations/_posts/foundations/config/0002-02-01-Config-Scopes-Visualization.md:7]: I liked this diagram. Why did it go away?

File: presentations/_posts/foundations/config/0003-01-01-User-Information.md [hunk 3]
```diff
@@ -1,8 +0,0 @@
----
-chapter: Config
-layout: slide
-tags: ['config']
----
-	git config --global user.name "GitHub Trainer"
-	git config --global user.email "training@github.com"
```
Comment (matthewmccullough) [inline presentations/_posts/foundations/config/0003-01-01-User-Information.md:7]: Where did the user config info go?

File: presentations/_posts/foundations/github/intro/0001-01-01-GitHub-Hosted-Repo.md [hunk 4]
```diff
@@ -2,7 +2,6 @@
 chapter: GitHub
 layout: slide
 tags: ['github/intro']
-categories: ['slidecontent']
```
Comment (matthewmccullough) [inline presentations/_posts/foundations/github/intro/0001-01-01-GitHub-Hosted-Repo.md:5]: Thanks for nuking that "dirt" I added.

File: presentations/_posts/foundations/github/intro/0002-01-01-Cloning-from-GitHub.md [hunk 5]
```diff
@@ -2,9 +2,12 @@
...
 
 Obtain a local "copy":
 
-	git clone git@github.com:<username>/<repository>.git
\ No newline at end of file
+	# SSH
+	git clone git@github.com:<username>/<repository>.git
+
+	# HTTPS
+	https://github.com/<username>/<repository>.git
```
Comment (matthewmccullough) [inline presentations/_posts/foundations/github/intro/0002-01-01-Cloning-from-GitHub.md:13]: Nice to have the other protocol examples

File: presentations/_posts/foundations/config/0002-02-01-Config-Scopes-Visualization.md (hunk 2 above)
Comment (jordanmccullough) [inline presentations/_posts/foundations/config/0002-02-01-Config-Scopes-Visualization.md:7]: Not sure, haven't dug through log to see. Either way, it's _back_!

File: presentations/_posts/foundations/config/0003-01-01-User-Information.md (hunk 3 above)
Comment (jordanmccullough) [inline presentations/_posts/foundations/config/0003-01-01-User-Information.md:7]: User info config slide should still be there. Just reordered the
examples to build up in a more logical flow. Please confirm so I'm not
overlooking a missing example.

File: presentations/_posts/foundations/log/0005-01-01-Additional-Options.md [hunk 6]
```diff
@@ -7,8 +7,5 @@ tags: ['log']
 	# Search for string match _in_ patch
 	git log -S <StringInPatch>
 
-	# Listing files of patch
-	git log --stat
```
Comment (matthewmccullough) [inline presentations/_posts/foundations/log/0005-01-01-Additional-Options.md:11]: I liked log --stat as another example. Did it migrate to its own slide?

File: presentations/_posts/foundations/config/0002-01-01-Scopes.md (hunk 1 above)
Comment (jordanmccullough) [inline presentations/_posts/foundations/config/0002-01-01-Scopes.md:17]: That totally works for me. Will do tomorrow and push an update to the
branch for the PR.

File: presentations/_posts/foundations/network/0007-01-01-Pull-vs-Fetch.md [hunk 7]
```diff
@@ -5,15 +5,18 @@ tags: ['network']
 categories: ['slidecontent']
 ---
 
-	git pull <remote>
+	git pull <remote> <branch>
 
-	git fetch <remote>
```
Comment (matthewmccullough) [inline presentations/_posts/foundations/network/0007-01-01-Pull-vs-Fetch.md:10]: I like having the branch-less invocations, at least as a supplement. For fetch, it tends to be the most typical invocation

File: presentations/_posts/foundations/log/0005-01-01-Additional-Options.md (hunk 6 above)
Comment (jordanmccullough) [inline presentations/_posts/foundations/log/0005-01-01-Additional-Options.md:11]: Yes. --stat was shown on two sequential slides so it was removed from the one.

File: presentations/_posts/foundations/network/0007-01-01-Pull-vs-Fetch.md (hunk 7 above)
Comment (jordanmccullough) [inline presentations/_posts/foundations/network/0007-01-01-Pull-vs-Fetch.md:10]: But what about using the remote + branch as a way of utilizing
FETCH_HEAD? Too much for Foundations class?

File: presentations/_posts/foundations/log/0005-01-01-Additional-Options.md (hunk 6 above)
Comment (matthewmccullough) [inline presentations/_posts/foundations/log/0005-01-01-Additional-Options.md:11]: Perfect
.

File: presentations/_posts/foundations/network/0007-01-01-Pull-vs-Fetch.md (hunk 7 above)
Comment (matthewmccullough) [inline presentations/_posts/foundations/network/0007-01-01-Pull-vs-Fetch.md:10]: Yeah. I really try not to bring FETCH_HEAD into foundations unless they just blast through everything (and only one class a year does that).

**Comment by matthewmccullough** [comment]
I'm good with merging this. What say you @jordanmccullough ?

**Comment by matthewmccullough** [comment]
It is a dramatic improvement, and we can always iterate again. Can't hold back a ship just because I want to tweak it more. These are all good.

File: presentations/_posts/foundations/config/0003-01-01-User-Information.md (hunk 3 above)
Comment (jordanmccullough) [inline presentations/_posts/foundations/config/0003-01-01-User-Information.md:7]: Reorder the config chapter. The name and email steps are now on slide 1 of 0001-01-01-User-Information.md

File: presentations/_posts/foundations/config/0002-01-01-Scopes.md (hunk 1 above)
Comment (jordanmccullough) [inline presentations/_posts/foundations/config/0002-01-01-Scopes.md:17]: Reordered. --System first, --global second, --local third.

**Comment by jordanmccullough** [comment]
Last adjustments done. This PR is ready to :ship: once we have @github/training-team eyes on it and agree the changes aren't too invasive.

**Comment by jordanmccullough** [comment]
@github/training-team Just made one final adjustment to the Foundations content, this time correcting a typo on the Merging chapter content.

This thing is ready to ship. Only hiccup is that the migration to the new _includes path-ing for notes is resulting in some merge conflicts. Advise how you'd like me to proceed and I'll do what it takes to get these improvements to content in and live.

**Comment by jordanmccullough** [comment]
Merged gh-pages into this branch and locally tested to resolve the merge conflict. This PR is ready for :rocket:

**Comment by matthewmccullough** [comment]
@jordanmccullough Nice technique there on the reverse merge to get the feature branch up to date.

**Comment by jordanmccullough** [comment]
:wink:

**Comment by tlberglund** [comment]
I would teach from this deck. :ship:ing.

**Comment by brntbeer** [comment]
Sorry for lagging on responding.  Loved this PR!

**Pull request merged**